app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: [dict], cafe: Cafe) -> None | str:
```

Comment:
There are no cases when `None` is returned here, leave only `str` in the return type annotation. Besides that, indicate list explicitly in the type annotation
```suggestion
def go_to_cafe(friends: list[dict], cafe: Cafe) -> None | str:
```